### DISCUSSION TITLE
Plugin: Request for help with 'Variant cannot be generated because selector contains no classes.'

### DISCUSSION THREAD

**Discussion by samtrion** [tailwindlabs/tailwindcss#3233] [Help] [answered]
Hi,

I am currently trying to write a plugin for List ::marker, however I am failing the tests. I have prepared a [Tailwind Play](https://play.tailwindcss.com/8I7kBs3osR?file=config), which gives the same error message.

**Comment by adamwathan** [comment, marked as answer]
Add a “.” in front of “list-marker”, right now it’s not a class. Should fix it.

**Reply by samtrion** [reply]
Sometimes it's so obvious that it's easy to miss.
